packages/react-email/package.json

```diff
@@ -3,14 +3,24 @@
   "version": "1.10.1-canary.0",
   "description": "A live preview of your emails right in your browser.",
   "bin": {
-    "email": "./dist/source/index.js"
+    "email": "./cli/index.js"
```

Comment:
Shouldn't this be pointing to the built directory?

Reply:
This is the new build directory, I built it with tsup, since we aren't building the next app as well I found this was the cleanest solution. 

The build process now generates a `./cli/index.js` to be used as binary.